projects/1.18/assets/tinkers-reforged/tinkers-reforged/lang/zh_cn.json

```
@@ -0,0 +1,332 @@
+  "item.tinkers_reforged.molten_magma_steel_bucket": "熔融岩浆钢桶",
+  "item.tinkers_reforged.molten_proto_lava_bucket": "熔融原始熔岩桶",
+  "item.tinkers_reforged.molten_qivium_bucket": "熔融熔钴桶",
+  "item.tinkers_reforged.molten_red_beryl_bucket": "熔融红绿柱石桶",
+  "item.tinkers_reforged.molten_redstone_bucket": "液态红石桶",
+  "item.tinkers_reforged.molten_shulker_bucket": "熔融潜影桶",
+  "item.tinkers_reforged.molten_wavy_bucket": "熔融波菌桶",
+  "item.tinkers_reforged.molten_yokel_bucket": "熔融海枷桶",
+  "item.tinkers_reforged.qivium_dust": "熔钴粉",
+  "item.tinkers_reforged.qivium_ingot": "熔钴锭",
+  "item.tinkers_reforged.qivium_nugget": "熔钴粒",
+  "item.tinkers_reforged.raw_aluminum": "粗铝土",
+  "item.tinkers_reforged.raw_kepu": "粗凯普",
+  "item.tinkers_reforged.red_beryl_gem": "红绿柱石",
+  "item.tinkers_reforged.reforging_guide": "再锻指南",
+  "item.tinkers_reforged.wavy_dust": "波菌粉",
+  "item.tinkers_reforged.wavy_ingot": "波菌锭",
+  "item.tinkers_reforged.wavy_nugget": "波菌粒",
+  "item.tinkers_reforged.yokel_dust": "海枷粉",
+  "item.tinkers_reforged.yokel_ingot": "海枷锭",
+  "item.tinkers_reforged.yokel_nugget": "海枷粒",
+  "itemGroup.tinkers_reforged": "工匠再锻丨物品",
+  "material.tinkers_reforged.aluminum": "铝",
+  "material.tinkers_reforged.aluminum.encyclopedia": "铝金属。",
+  "material.tinkers_reforged.aluminum.flavor": "坚固耐用。",
+  "material.tinkers_reforged.baolian": "宝莲",
+  "material.tinkers_reforged.baolian.encyclopedia": "由黑曜石和红磷锰石制成。",
+  "material.tinkers_reforged.baolian.flavor": "嗯，宝石组合？",
+  "material.tinkers_reforged.blazing_copper": "烈焰铜",
+  "material.tinkers_reforged.blazing_copper.encyclopedia": "如果烈焰血和铜有一个孩子，会怎么样？",
+  "material.tinkers_reforged.blazing_copper.flavor": "炽热的温度！",
+  "material.tinkers_reforged.chorus_metal": "紫颂金属",
+  "material.tinkers_reforged.chorus_metal.encyclopedia": "紫颂和潜影合金。",
+  "material.tinkers_reforged.chorus_metal.flavor": "紫颂和潜影合金。",
+  "material.tinkers_reforged.crusteel": "地壳钢",
+  "material.tinkers_reforged.crusteel.encyclopedia": "下界铜合金。",
+  "material.tinkers_reforged.crusteel.flavor": "下界地壳。",
+  "material.tinkers_reforged.cyber_steel": "网络钢",
+  "material.tinkers_reforged.cyber_steel.encyclopedia": "钢遇见紫颂金属。",
+  "material.tinkers_reforged.cyber_steel.flavor": "一个很酷的末地合金。",
+  "material.tinkers_reforged.duralumin": "硬铝",
+  "material.tinkers_reforged.duralumin.encyclopedia": "铝合金。",
+  "material.tinkers_reforged.duralumin.flavor": "最早的时效硬化铝合金之一。",
+  "material.tinkers_reforged.durasteel": "耐钢",
+  "material.tinkers_reforged.durasteel.encyclopedia": "末影和下界合金的混合体。",
+  "material.tinkers_reforged.durasteel.flavor": "等等，这不是矿石吗？",
+  "material.tinkers_reforged.electrical_copper": "电工铜",
+  "material.tinkers_reforged.electrical_copper.encyclopedia": "被电化的铜。",
+  "material.tinkers_reforged.electrical_copper.flavor": "它并不像人们想象的那样有电。",
+  "material.tinkers_reforged.ender_bone": "末影之骨",
+  "material.tinkers_reforged.ender_bone.encyclopedia": "骨头，但有更多的耐久。",
+  "material.tinkers_reforged.ender_bone.flavor": "骨头与末影混合。",
```

```suggestion
  "material.tinkers_reforged.ender_bone.flavor": "骨头与末影混合",
```
后续不再一一提出
匠魂语言文件本身已经做出改动，flavor和description（材料的）后面无实际表意的句号删去
推荐同样做出改动